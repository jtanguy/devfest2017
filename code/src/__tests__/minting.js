const simple = require('../simple');

describe('Minting Macaroons', () => {

  test('m', () => {
    expect(simple.m.inspect()).toMatchSnapshot();
  });

  test('m2 has two caveats', () => {
    expect(simple.m2.inspect()).toMatchSnapshot();
  });

});

