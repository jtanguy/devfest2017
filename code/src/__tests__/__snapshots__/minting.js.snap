// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Minting Macaroons m 1`] = `
"location ncrafts.io
identifier it's a me, Julien!
signature 33241e6038895a6dee3b840cd800c9c1f209b620ae3e972a0d674f487179a54f
"
`;

exports[`Minting Macaroons m2 has two caveats 1`] = `
"location ncrafts.io
identifier it's a me, Julien!
cid talk = macaroons-hands-on
cid time < 2017-05-18T14:45+02:00
signature 77f27c4442cc89e99cfaf966c32b3d168392366a977d4fb3464a3894ca411c9b
"
`;
